packages/client/src/App.tsx

```diff
@@ -327,13 +402,19 @@ export function App(props: IAppProps) {
                                             />
                                             <ProtectedRoute
                                               exact
+                                              scopes={[
+                                                SCOPES.RECORD_REGISTRATION_CORRECT
+                                              ]}
                                               path={
                                                 routes.CERTIFICATE_CORRECTION
                                               }
                                               component={CorrectionForm}
                                             />
                                             <ProtectedRoute
                                               exact
+                                              scopes={[
+                                                SCOPES.RECORD_REGISTRATION_CORRECT
```

Comment:
```suggestion
                                                SCOPES.RECORD_REGISTRATION_REQUEST_CORRECTION
```
I know this is a bit confusing but CERTIFICATE_CORRECTION_REVIEW is actually a part of the request correction form